spec/lib/http_spec.rb

```diff
@@ -430,6 +430,31 @@ def setsockopt(*args)
         expect(response.to_s).to eq("#{body}-deflated")
       end
     end
+
+    context "with :normalize_uri" do
+      module Normalizer
+        def self.call(uri)
+          uri
+        end
+      end
+
+      it "Use the defaul Uri normalizer when user does not use uri normalizer" do
+        response = HTTP.get "#{dummy.endpoint}/hello world"
+        expect(response.to_s).to eq("hello world")
+      end
+
+      it "Use the custom Uri Normalizer method" do
+        client = HTTP.use(:normalize_uri => {:normalizer => Normalizer})
```

Comment:
As I said above you don't need `Normalizer` module here. Replace this line with:

```suggestion
        client = HTTP.use(:normalize_uri => {:normalizer => :itself.to_proc})
```